temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -205,3 +206,26 @@ message DeleteNamespaceResponse {
     // The async operation.
     temporal.api.cloud.operation.v1.AsyncOperation async_operation = 1;
 }
+
+message GetRegionsRequest {
+    // The requested size of the page to retrieve - optional.
+    // Cannot exceed 1000. Defaults to 100. 
+    int32 page_size = 1;
+    // The page token if this is continuing from another response - optional.
+    string page_token = 2;
+}
+
+message GetRegionsResponse {
+    // The temporal cloud regions.
+    repeated temporal.api.cloud.region.v1.Region regions = 1;
+}
+
+message GetRegionRequest {
+    // The id of the region to get.
+    string region = 1;
+}
+
+message GetRegionResponse {
+    // The temporal cloud region.
+    temporal.api.cloud.region.v1.Region region = 1;
+}
```

Comment:
I'm not seeing where these messages are ever used

It might be best, before merging APIs, that the implementation of them also be ready to merge. Often during implementation you find adjustments you need to the APIs. Impl and API should both remain in development while they are in development and then both get merged when they are ready.

Reply:
Added the service entry. 

Absolutely I do not plan on merging these changes into the main branch till I have the implementation ready and deployed on prod.